Practic/Practic/DataTable.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Practic
+{
+    public class DataTable
+    {
+        public string Lable { get; set; }
+        public int Sum { get; set; }
+
+        public DataTable(string lable, int sum)
+        {
+            Lable = lable;
+            Sum = sum;
+
+        }
+
+        public static List<DataTable> CreateTableWithOutSerialize(string name)
+        {                    
+            FileStream fileStream = File.Open(name, FileMode.OpenOrCreate);
+
+            StreamReader streamReader = new StreamReader(fileStream);
+
+            JsonSerializer js = new JsonSerializer();
+
+            var b = (List<DataTable>) js.Deserialize(streamReader, typeof (List<DataTable>));
+            return b;
+
+        }
+        public static string GetTextFromFile(string JsonData)
```

Comment:
JsonData name is confusing because it is not data it is name of the file with data
JsonDataFileName would be better